packages/mobile-top-nav/src/data/menus.js

```diff
@@ -0,0 +1,337 @@
+const texts = {
+  heading: 'Books',
+  iconLinks: [{
+    title: 'Books to Borrow',
+    icon: 'https://archive.org/images/book-lend.png',
+    url: 'https://archive.org/details/inlibrary?sort=-publicdate',
```

Comment:
Two thoughts:
1. I would package the icons in the repo so we're not relying on the production web server to serve up the icons.
2. I don't think we can use absolute URLs here, otherwise we can't use it in a review app.

Reply:
Yeah, there's a few spots like this. Ultimately, this menus file will be replaced with a fetch for the actual data to render the subnavs, so I figured in the short term this would be the least amount of cleanup.